includes/class-sensei-usage-tracking.php

```diff
@@ -257,6 +257,40 @@ private function get_wccom_extensions() {
 		return $wccom_extensions;
 	}
 
+	/**
+	 * Get the template override data.
+	 *
+	 * @return array
+	 */
+	public function get_template_data() {
+		$theme              = wp_get_theme();
+		$template_overrides = Sensei_Status::instance()->get_template_override_status();
+
+		$data = [
+			'version'       => Sensei()->version,
+			'theme'         => $theme['Name'],
+			'theme_version' => $theme['Version'],
+			'templates'     => count( $template_overrides ),
+			'mismatch'      => 0,
+		];
+
+		foreach ( $template_overrides as $template => $versions ) {
+			$property_key          = preg_replace( '/[^0-9_a-z]/', '_', strtr( strtolower( $template ), [ '.php' => '' ] ) );
```

Comment:
I find this line and the next one a bit hard to read. Can we explain a bit in a comment what `$template` and `$versions` are and how we modify them?

Reply:
@gkaragia  Added docs in e03fef4a912dd595f679985857475dab8945db3f